sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationOperation.cs

```diff
@@ -0,0 +1,502 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.AI.DocumentTranslation.Models;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.AI.DocumentTranslation
+{
+    /// <summary> The DocumentTranslationOperation class for LRO. </summary>
+    public class DocumentTranslationOperation : PageableOperation<DocumentStatusDetail>
+    {
+        /// <summary>Provides communication with the Translator Cognitive Service through its REST API.</summary>
+        private readonly DocumentTranslationRestClient _serviceClient;
+
+        /// <summary>Provides tools for exception creation in case of failure.</summary>
+        private readonly ClientDiagnostics _diagnostics;
+
+        /// <summary>
+        /// Operation created date time.
+        /// </summary>
+        public DateTimeOffset CreatedOn => _createdOn;
+
+        /// <summary>
+        /// Date time in which the operation&apos;s status has been updated.
+        /// </summary>
+        public DateTimeOffset LastModified => _lastModified;
+
+        /// <summary>
+        /// List of possible statuses for job or document.
+        /// </summary>
+        public TranslationStatus Status => _status;
+
+        /// <summary>
+        /// Total number of documents in the operation.
+        /// </summary>
+        public int DocumentsTotal => _documentsTotal;
+
+        /// <summary>
+        /// Number of documents failed to translate in the operation.
+        /// </summary>
+        public int DocumentsFailed => _documentsFailed;
+
+        /// <summary>
+        /// Number of documents translated successfully in the operation.
```

Comment:
do we need the `in the operation` for all these properties?
Maybe just `Number of documents translated successfully`?